com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/RenderPass/MotionVectors/CameraMotionVectors.shader

```diff
@@ -57,6 +57,13 @@ Shader "Hidden/HDRP/CameraMotionVectors"
 
             // Convert from Clip space (-1..1) to NDC 0..1 space
             float2 motionVector = (positionCS - previousPositionCS);
+
+#ifdef KILL_MICRO_MOVEMENT
+            // We multiply by 2 as the motion vectors have been already converted to NDC when we reach this function.
```

Comment:
I don't see the *2 described in the comment. Mistake or outdated comment? (same above)

Reply:
Leftover from older code, removed!